.github/workflows/cyberduck-release-windows.yml

```diff
@@ -0,0 +1,208 @@
+name: cyberduck-release-windows
+on:
+  workflow_dispatch:
+    inputs:
+      GIT_TAG:
+        required: false
+
+env:
+  AWS_ACCESS_KEY_DEPLOYMENT: "${{ secrets.AWS_ACCESS_KEY_DEPLOYMENT }}"
+  AWS_SECRET_KEY_DEPLOYMENT: "${{ secrets.AWS_SECRET_KEY_DEPLOYMENT }}"
+  AWS_CUSTOM_IP5_ACCESS_KEY_DEPLOYMENT: "${{ secrets.AWS_CUSTOM_IP5_ACCESS_KEY_DEPLOYMENT }}"
+  AWS_CUSTOM_IP5_SECRET_KEY_DEPLOYMENT: "${{ secrets.AWS_CUSTOM_IP5_SECRET_KEY_DEPLOYMENT }}"
+  RACKSPACE_USER: "${{ secrets.RACKSPACE_USER }}"
+  RACKSPACE_PASSWORD: "${{ secrets.RACKSPACE_PASSWORD }}"
+
+jobs:
+  build:
+    runs-on:
+      - windows-latest
+    steps:
+    - name: clean workspace
+      shell: ruby {0}
+      run: |-
+        require "fileutils"
+        Dir.chdir(ENV["GITHUB_WORKSPACE"]) do
+          paths = Dir.glob(["**/*"])
+          paths -= Dir.glob([".git/**", ".repository/**"])
+          paths.each do |path|
+            File.delete(path) if File.file?(path)
+            FileUtils.rm_rf(path) if File.directory?(path)
+          end
+        end
+
+    - name: checkout
+      uses: actions/checkout@v4.1.0
+      with:
+        fetch-depth: '0'
+        ref: ${{ inputs.GIT_TAG }}
```

Comment:
Don't think this input is required when the user selects a tag for the run from _Run workflow_.